src/ui/FileViewer/Slider.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { FileViewerComponentProps, ViewerTypes } from './types';
+import Icon, { IconColors, IconTypes } from '../Icon';
+
+const BUTTON_ICON_SIDE_LENGTH = '32px';
```

Comment:
How about using this place for constances?
https://github.com/sendbird/sendbird-uikit-react/blob/main/src/utils/consts.ts